NEWS.md

```diff
@@ -15,6 +15,9 @@ for human readability.
 - New time integrator `PairedExplicitRK2`, implementing the second-order paired explicit Runge-Kutta
   method with [Convex.jl](https://github.com/jump-dev/Convex.jl) and [ECOS.jl](https://github.com/jump-dev/ECOS.jl) ([#1908])
 - Add subcell limiting support for `StructuredMesh` ([#1946]).
+- New time integrator `PairedExplicitRK3`, implementing the third-order paired explicit Runge-Kutta
+  method with [Convex.jl](https://github.com/jump-dev/Convex.jl), [ECOS.jl](https://github.com/jump-dev/ECOS.jl),
+  and [NLsolve.jl](https://github.com/JuliaNLSolvers/NLsolve.jl) ([TBD])
```

Comment:
```suggestion
  and [NLsolve.jl](https://github.com/JuliaNLSolvers/NLsolve.jl)
```

Am I missing something or is this obsolete?

Reply:
Oh, TBD is a placeholder for the PR number

Reply:
Ah makes sense. I thought this is also a package link, my bad.